xcodeproj/internal/xcodeproj_rule.bzl

```diff
@@ -1240,6 +1244,12 @@ transitive dependencies of the targets specified in the
             allow_single_file = True,
             default = Label("//xcodeproj/internal:installer.template.sh"),
         ),
+        "_swiftc_stub": attr.label(
+            # Not "exec", because it's really for the host machine
+            cfg = "host",
```

Comment:
"exec" is also correct though, and "host" is going to be removed from bazel.

Reply:
This will break workspaces that have [`--incompatible_disable_starlark_host_transitions`](https://github.com/bazelbuild/bazel/blob/0d0df1951a7a9a7744028054f83386a08cf83e5f/src/main/java/com/google/devtools/build/lib/packages/semantics/BuildLanguageOptions.java#L551-L560) set.